pkg/extensions/search/cve/trivy/scanner_test.go

```diff
@@ -24,6 +25,45 @@ import (
 	"zotregistry.io/zot/pkg/test/mocks"
 )
 
+func TestMain(t *testing.T) {
```

Comment:
What is this function testing besides what is already tested?

Reply:
I've updated the name. Since I've removed zot-test from pretty much every test, it's not scanned anymore and there was a line that was missing from coverage, it's related to the output of Trivy's scan report. To be sure everything was as before, i've added this scan separately.